config/local_info.xml

```diff
@@ -1,9 +1,80 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema>
+<config> 
+        <!-- 
+          ## Begin local override.
+          ## Specify a url attribute to be matched  against the incoming requestor
+          ## url. Only a single url is supported.
```

Comment:
> Only a single url is supported.

Does that mean only one `<local_info url=X>...</local_info>` block can be defined? 
Or does it mean you can't do things like this `<local_info url=X, Y>...</local_info>`

Reply:
No. Multiple local_info element blocks are allowed. And yes, you can't specifiy multiple values per url attribute.